### PR TITLE
Instructions for installing a specific CLI version

### PR DESCRIPTION
**Pull request by berndverst**
Useful for installing RCs

Signed-off-by: Bernd Verst <github@bernd.dev>

### CODE REVIEW

File: daprdocs/content/en/getting-started/install-dapr-cli.md [hunk 1]
```diff
@@ -29,6 +29,15 @@ Install the latest Linux Dapr CLI to `/usr/local/bin`:
 wget -q https://raw.githubusercontent.com/dapr/cli/master/install/install.sh -O - | /bin/bash
 ```
 
+##### Installing a specific CLI version
+
+This example shows how to install CLI version `1.9.1`. Release candidates can be installed similarly, simplify specify the version, e.g. `1.10.0-rc.3`
```
Comment (berndverst) [inline daprdocs/content/en/getting-started/install-dapr-cli.md]:
```suggestion
This example shows how to install CLI version `1.9.1`. Release candidates can be installed similarly, simply specify the version, e.g. `1.10.0-rc.3`
```

File: daprdocs/content/en/getting-started/install-dapr-cli.md [hunk 2]
```diff
@@ -37,6 +46,14 @@ If you do not have access to the `sudo` command or your username is not in the `
 wget -q https://raw.githubusercontent.com/dapr/cli/master/install/install.sh -O - | DAPR_INSTALL_DIR="$HOME/dapr" /bin/bash
 ```
 
+##### Installing a specific CLI version without `sudo`
+
+This example shows how to install CLI version `1.9.1`. Release candidates can be installed similarly, simplify specify the version, e.g. `1.10.0-rc.3`
```
Comment (berndverst) [inline daprdocs/content/en/getting-started/install-dapr-cli.md]:
```suggestion
This example shows how to install CLI version `1.9.1`. Release candidates can be installed similarly, simply specify the version, e.g. `1.10.0-rc.3`
```

**Review by philliphoff — APPROVED** [review]
I think it could use similar updates on the Mac section (as the code tabs will tend to hide it), but just having it in the docs goes a long way in making it easier to try out new versions.

**Comment by berndverst** [comment]
> I think it could use similar updates on the Mac section (as the code tabs will tend to hide it), but just having it in the docs goes a long way in making it easier to try out new versions.

You are too late. This was done before you left this comment. ☺️

**Comment by hhunter-ms** [comment]
@berndverst I'm pretty sure I fixed the broken link in the submodule PR I just merged, if you can pull it in

File: daprdocs/content/en/getting-started/install-dapr-cli.md [hunk 3]
```diff
@@ -29,6 +29,15 @@ Install the latest Linux Dapr CLI to `/usr/local/bin`:
 wget -q https://raw.githubusercontent.com/dapr/cli/master/install/install.sh -O - | /bin/bash
 ```
 
+##### Installing a specific CLI version
+
+This example shows how to install CLI version `1.9.1`. Release candidates can be installed similarly, simply specify the version, e.g. `1.10.0-rc.3`.
```
Comment (hhunter-ms) [inline daprdocs/content/en/getting-started/install-dapr-cli.md]:
```suggestion
The following example shows how to install CLI version `1.9.1`. You can also install release candidates by specifying the version (for example, `1.10.0-rc.3`).
```

File: daprdocs/content/en/getting-started/install-dapr-cli.md [hunk 4]
```diff
@@ -37,6 +46,14 @@ If you do not have access to the `sudo` command or your username is not in the `
 wget -q https://raw.githubusercontent.com/dapr/cli/master/install/install.sh -O - | DAPR_INSTALL_DIR="$HOME/dapr" /bin/bash
 ```
 
+##### Installing a specific CLI version without `sudo`
+
+This example shows how to install CLI version `1.9.1`. Release candidates can be installed similarly, simply specify the version, e.g. `1.10.0-rc.3`.
```
Comment (hhunter-ms) [inline daprdocs/content/en/getting-started/install-dapr-cli.md]:
```suggestion
The following example shows how to install CLI version `1.9.1`. You can also install release candidates by specifying the version (for example, `1.10.0-rc.3`).
```

File: daprdocs/content/en/getting-started/install-dapr-cli.md [hunk 5]
```diff
@@ -51,6 +68,14 @@ powershell -Command "iwr -useb https://raw.githubusercontent.com/dapr/cli/master
 
 **Note:** Updates to PATH might not be visible until you restart your terminal application.
 
+##### Installing a specific CLI version
+
+This example shows how to install CLI version `1.9.1`. Release candidates can be installed similarly, simply specify the version, e.g. `1.10.0-rc.3`.
```
Comment (hhunter-ms) [inline daprdocs/content/en/getting-started/install-dapr-cli.md]:
```suggestion
The following example shows how to install CLI version `1.9.1`. You can also install release candidates by specifying the version (for example, `1.10.0-rc.3`).
```

File: daprdocs/content/en/getting-started/install-dapr-cli.md [hunk 6]
```diff
@@ -60,6 +85,15 @@ $Env:DAPR_INSTALL_DIR = "<your_alt_install_dir_path>"
 $script=iwr -useb https://raw.githubusercontent.com/dapr/cli/master/install/install.ps1; $block=[ScriptBlock]::Create($script); invoke-command -ScriptBlock $block -ArgumentList "", "$Env:DAPR_INSTALL_DIR"
 ```
 
+#### Installing a specific CLI version without administrative rights
+
+This example shows how to install CLI version `1.9.1`. Release candidates can be installed similarly, simply specify the version, e.g. `1.10.0-rc.3`.
```
Comment (hhunter-ms) [inline daprdocs/content/en/getting-started/install-dapr-cli.md]:
```suggestion
The following example shows how to install CLI version `1.9.1`. You can also install release candidates by specifying the version (for example, `1.10.0-rc.3`).
```

File: daprdocs/content/en/getting-started/install-dapr-cli.md [hunk 7]
```diff
@@ -97,6 +131,14 @@ Install the latest Darwin Dapr CLI to `/usr/local/bin`:
 curl -fsSL https://raw.githubusercontent.com/dapr/cli/master/install/install.sh | /bin/bash
 ```
 
+##### Installing a specific CLI version
+
+This example shows how to install CLI version `1.9.1`. Release candidates can be installed similarly, simply specify the version, e.g. `1.10.0-rc.3`.
```
Comment (hhunter-ms) [inline daprdocs/content/en/getting-started/install-dapr-cli.md]:
```suggestion
The following example shows how to install CLI version `1.9.1`. You can also install release candidates by specifying the version (for example, `1.10.0-rc.3`).
```

File: daprdocs/content/en/getting-started/install-dapr-cli.md [hunk 8]
```diff
@@ -130,6 +172,14 @@ If you do not have access to the `sudo` command or your username is not in the `
 curl -fsSL https://raw.githubusercontent.com/dapr/cli/master/install/install.sh | DAPR_INSTALL_DIR="$HOME/dapr" /bin/bash
 ```
 
+##### Installing a specific CLI version without `sudo`
+
+This example shows how to install CLI version `1.9.1`. Release candidates can be installed similarly, simply specify the version, e.g. `1.10.0-rc.3`.
```
Comment (hhunter-ms) [inline daprdocs/content/en/getting-started/install-dapr-cli.md]:
```suggestion
The following example shows how to install CLI version `1.9.1`. You can also install release candidates by specifying the version (for example, `1.10.0-rc.3`).
```

**Review by hhunter-ms — CHANGES_REQUESTED** [review]
quick pass

**Review by hhunter-ms — APPROVED** [review]

File: daprdocs/content/en/getting-started/install-dapr-cli.md [hunk 9]
```diff
@@ -29,6 +29,15 @@ Install the latest Linux Dapr CLI to `/usr/local/bin`:
 wget -q https://raw.githubusercontent.com/dapr/cli/master/install/install.sh -O - | /bin/bash
 ```
 
+##### Installing a specific CLI version
+
+The following example shows how to install CLI version `1.9.1`. You can also install release candidates by specifying the version (for example, `1.10.0-rc.3`).
```
Comment (hhunter-ms) [inline daprdocs/content/en/getting-started/install-dapr-cli.md]:
```suggestion
The following example shows how to install CLI version `{{% dapr-latest-version cli="true" %}}`. You can also install release candidates by specifying the version (for example, `1.10.0-rc.3`).
```

File: daprdocs/content/en/getting-started/install-dapr-cli.md [hunk 10]
```diff
@@ -37,6 +46,14 @@ If you do not have access to the `sudo` command or your username is not in the `
 wget -q https://raw.githubusercontent.com/dapr/cli/master/install/install.sh -O - | DAPR_INSTALL_DIR="$HOME/dapr" /bin/bash
 ```
 
+##### Installing a specific CLI version without `sudo`
+
+The following example shows how to install CLI version `1.9.1`. You can also install release candidates by specifying the version (for example, `1.10.0-rc.3`).
```
Comment (hhunter-ms) [inline daprdocs/content/en/getting-started/install-dapr-cli.md]:
```suggestion
The following example shows how to install CLI version `{{% dapr-latest-version cli="true" %}}`. You can also install release candidates by specifying the version (for example, `1.10.0-rc.3`).
```

File: daprdocs/content/en/getting-started/install-dapr-cli.md [hunk 11]
```diff
@@ -51,6 +68,14 @@ powershell -Command "iwr -useb https://raw.githubusercontent.com/dapr/cli/master
 
 **Note:** Updates to PATH might not be visible until you restart your terminal application.
 
+##### Installing a specific CLI version
+
+The following example shows how to install CLI version `1.9.1`. You can also install release candidates by specifying the version (for example, `1.10.0-rc.3`).
```
Comment (hhunter-ms) [inline daprdocs/content/en/getting-started/install-dapr-cli.md]:
```suggestion
The following example shows how to install CLI version `{{% dapr-latest-version cli="true" %}}`. You can also install release candidates by specifying the version (for example, `1.10.0-rc.3`).
```

File: daprdocs/content/en/getting-started/install-dapr-cli.md [hunk 12]
```diff
@@ -60,6 +85,15 @@ $Env:DAPR_INSTALL_DIR = "<your_alt_install_dir_path>"
 $script=iwr -useb https://raw.githubusercontent.com/dapr/cli/master/install/install.ps1; $block=[ScriptBlock]::Create($script); invoke-command -ScriptBlock $block -ArgumentList "", "$Env:DAPR_INSTALL_DIR"
 ```
 
+#### Installing a specific CLI version without administrative rights
+
+The following example shows how to install CLI version `1.9.1`. You can also install release candidates by specifying the version (for example, `1.10.0-rc.3`).
```
Comment (hhunter-ms) [inline daprdocs/content/en/getting-started/install-dapr-cli.md]:
```suggestion
The following example shows how to install CLI version `{{% dapr-latest-version cli="true" %}}`. You can also install release candidates by specifying the version (for example, `1.10.0-rc.3`).
```

File: daprdocs/content/en/getting-started/install-dapr-cli.md [hunk 13]
```diff
@@ -97,6 +131,14 @@ Install the latest Darwin Dapr CLI to `/usr/local/bin`:
 curl -fsSL https://raw.githubusercontent.com/dapr/cli/master/install/install.sh | /bin/bash
 ```
 
+##### Installing a specific CLI version
+
+The following example shows how to install CLI version `1.9.1`. You can also install release candidates by specifying the version (for example, `1.10.0-rc.3`).
```
Comment (hhunter-ms) [inline daprdocs/content/en/getting-started/install-dapr-cli.md]:
```suggestion
The following example shows how to install CLI version `{{% dapr-latest-version cli="true" %}}`. You can also install release candidates by specifying the version (for example, `1.10.0-rc.3`).
```

File: daprdocs/content/en/getting-started/install-dapr-cli.md [hunk 14]
```diff
@@ -130,6 +172,14 @@ If you do not have access to the `sudo` command or your username is not in the `
 curl -fsSL https://raw.githubusercontent.com/dapr/cli/master/install/install.sh | DAPR_INSTALL_DIR="$HOME/dapr" /bin/bash
 ```
 
+##### Installing a specific CLI version without `sudo`
+
+The following example shows how to install CLI version `1.9.1`. You can also install release candidates by specifying the version (for example, `1.10.0-rc.3`).
```
Comment (hhunter-ms) [inline daprdocs/content/en/getting-started/install-dapr-cli.md]:
```suggestion
The following example shows how to install CLI version `{{% dapr-latest-version cli="true" %}}`. You can also install release candidates by specifying the version (for example, `1.10.0-rc.3`).
```

**Pull request merged**